src/Credentials/UserRefreshCredentials.php

```diff
@@ -42,17 +46,21 @@ class UserRefreshCredentials extends CredentialsLoader
    *
    * @param array $jsonKey JSON credentials.
    *
-   * @param string $jsonKeyPath the path to a file containing JSON credentials.  If
-   *   jsonKey is set, it is ignored.
+   * @param string|array $jsonKey JSON credential file path or JSON credentials
+   *   as an associative array
    */
   public function __construct(
     $scope,
```

Comment:
Also, as an FYI scope is not used to refresh the access token, but seems like you guys are using it for the cache key. Not sure if there's a different way you want to do this, since for people not using the cache stuff, they'd just be passing in `null` for the scope, or if they pass in something, the scope is a no-op.